### PR TITLE
Updated missing image links and incorrect project link

### PR DESCRIPTION
**Pull request by amalraj28**
I have updated the image links that were outdated. In addition, I noticed that `projectLink` property of `aprendes-go-com-testes` was incorrectly written as githubLink, and tags of `Servo` were kept as a single string, instead of keeping each tag as separate string. Both these mistakes are corrected.

### CODE REVIEW

**Review by StackOverflowIsBetterThanAnyAI — APPROVED** [review]
well spotted

**Comment by amalraj28** [comment]
Thank you!

**Comment by amalraj28** [comment]
Hi @StackOverflowIsBetterThanAnyAI,

I just wanted to follow up on this PR. Is there anything else you need from my side before it can be merged?

Thank you!

**Comment by StackOverflowIsBetterThanAnyAI** [comment]
Hey, I think that the PR is absolutelly fine.
Unfortunatelly, I am not an owner of this repository, so I don't have write access which means that I cannot merge your PR.

**Comment by amalraj28** [comment]
Thank you for your prompt feedback. Could you please advise on whom I should contact to get the PR merged?

**Comment by ekrishnachaitanya2004** [comment]
I would like to extend an invitation to individuals interested in contributing to my open-source website. Your participation and contributions will be greatly appreciated.
[**Link**](https://github.com/Snakes-n-Networks/Dev-blogs.git)

**Pull request closed without merging**
